modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -1093,6 +1093,12 @@ definition {
 		}
 	}
 
+	macro viewPredefinedValueDropdownContent {
+		AssertElementPresent(
+			key_text = "${key_textContent}",
```

Comment:
@piresdm, please rename `key_text` to `key_textContent` to keep naming consistent. :wink:

Reply:
@natocesarrego done!